### PR TITLE
Create Intro-to-Cloud-Native.md

### PR DESCRIPTION
**Pull request by AmitKarnam**
Started to work on the Introductory reading materials , required for new comers in the community . 
Added notes on introduction to cloud native.

### CODE REVIEW

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Introduction to Cloud Native #
+
+### **What is Cloud Native?**  <hr/>
+ 
+***The Cloud Native Computing Foundation(CNCF)***  defines Cloud Native as technologies empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach.
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:5]:
```diff
- defines Cloud Native as technologies empower 
+ defines Cloud Native as technologies that empower 
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+1) Cloud-native applications are different from the traditional applications that run in our data centres. The applications that are designed in the traditional way are not built keeping cloud compatibility in mind, this hinders the smooth functioning of the application.<br/><br/>
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:7]:
```diff
-  data centres
+ data centers
- that are designed in the traditional way
+ that are designed traditionally
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+2) So, we need a new architecture for our applications to utilize the benefits of cloud. There is a need to design the applications keeping cloud in mind and take advantage of several services the cloud provides.<br/>
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:8]:
```diff
- cloud
+ the cloud
```
Note:- There are 2 "cloud" which needs to be replaced

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+### **Why Cloud Native?**   <hr/>
+Some of the various service included with cloud :
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:11]:
```diff
- service included with cloud
+ services included with a cloud 
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+***The Cloud Native Computing Foundation(CNCF)***  defines Cloud Native as technologies empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach.
+
+1) Cloud-native applications are different from the traditional applications that run in our data centres. The applications that are designed in the traditional way are not built keeping cloud compatibility in mind, this hinders the smooth functioning of the application.<br/><br/>
+2) So, we need a new architecture for our applications to utilize the benefits of cloud. There is a need to design the applications keeping cloud in mind and take advantage of several services the cloud provides.<br/>
+
+### **Why Cloud Native?**   <hr/>
+Some of the various service included with cloud : 
+- Isolation
+- Scalability
+- Value added cloud services
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:14]:
```diff
- Value added
+ Value-added
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+### **Why Cloud Native?**   <hr/>
+Some of the various service included with cloud : 
+- Isolation
+- Scalability
+- Value added cloud services 
+<br/>
+
+***Isolation***
+1) Isolation refers to the fact that a small part of rather large application can be built in isolation , and many such isolated parts together constitute of the application. These isolated parts of the application communicate over the network, most commonly using API's.
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:18]:
```diff
- part of rather large application can be built in isolation , and many such isolated parts together constitute of the application. These isolated parts of the application communicate over the network, most commonly using API's.
+ part of a rather large application can be built in isolation, and many such isolated parts together constitute the application. These isolated parts of the application communicate over the network, most commonly using an API.
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+2) The ability to create parts ( microservices ) of a large application in isolation helps in many factors of development and operations for example scalability,debugging etc.
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:19]:
```diff
- scalability,debugging etc.
+  scalability, debugging, etc.
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+3) So for example, in the case, when there is a bug or error in the application, you know which component to fix and this also avoids any side effects as the components are all unrelated pieces of code.
+
+***Scalability***
+1) Simply deploying your application on cloud does not make it cloud-native. To be cloud native it should be able to take full benefits of the cloud. One of the key features would be Scalability.
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:23]:
```diff
- Simply deploying your application on cloud does not make it cloud-native. To be cloud native it should be able to take full benefits of the cloud. One of the key features would be Scalability.
+ Simply deploying your application on the cloud does not make it cloud-native. To be cloud-native it should be able to take full benefits of the cloud. One of the key features would be Scalability.
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+2) In today's dynamic word it is not quite enough for the application to just perform as intended, but it should be able to handle the fluctuating traffic it recieves, i.e. **Ability to handle sudden increase in traffic**, In such cases the application should be able to perform well without breaking down.
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:24]:
```diff
- word 
+ world
-  recieves, i.e.  Ability to handle sudden increase in traffic, In such cases the application
+ receives, i.e. Ability to handle a sudden increase in traffic, In such cases, the application
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+3) To achieve this, cloud native application runs in multiple runtimes spread across multiple hosts. The applications should be designed and architected in a way that they support multi regional, active-active deployments.
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:25]:
```diff
- To achieve this, cloud native application runs in multiple runtimes spread across multiple hosts. The applications should be designed and architected in a way that they support multi regional, active-active deployments.
+ To achieve this, a cloud-native application runs in multiple runtimes spread across multiple hosts. The applications should be designed and architected in a way that supports multi-regional, active-active deployments.
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+ 
+***Value Addition***
+1) By using the value added services provided by the cloud provider, we will get to know all the available options on the cloud and we can also learn about all the new services. This will help us to take good long-termed decisions.
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:28]:
```diff
- value added 
+ value-added
```

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+2) We can reduce the operational risk and get things done at greater velocity by leveraging the value-added services that are available on the cloud.
+
+***RESOURCES***
+- [IBM's introduction to cloud native article](https://ibm.github.io/cloud-enterprise-examples/concepts/cloud-native-overview/)
+- [CNCF's What is cloud native and why does it exists](https://youtu.be/d_8Vly4_ofo)
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:33]:
```diff
- CNCF's What is cloud native and why does it exists
+ CNCF's What is cloud-native and why does it exist
```
I loved it :) Keep growing Amit

File: Reading-Materials/Intro-to-Cloud-Native.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+2) In today's dynamic word it is not quite enough for the application to just perform as intended, but it should be able to handle the fluctuating traffic it recieves, i.e. **Ability to handle sudden increase in traffic**, In such cases the application should be able to perform well without breaking down.
+3) To achieve this, cloud native application runs in multiple runtimes spread across multiple hosts. The applications should be designed and architected in a way that they support multi regional, active-active deployments.
+ 
+***Value Addition***
+1) By using the value added services provided by the cloud provider, we will get to know all the available options on the cloud and we can also learn about all the new services. This will help us to take good long-termed decisions.
+2) We can reduce the operational risk and get things done at greater velocity by leveraging the value-added services that are available on the cloud.
+
+***RESOURCES***
+- [IBM's introduction to cloud native article](https://ibm.github.io/cloud-enterprise-examples/concepts/cloud-native-overview/)
```
Comment (helper-uttam) [inline Reading-Materials/Intro-to-Cloud-Native.md:32]:
```diff
- IBM's introduction to cloud native article
+ IBM's introduction to cloud-native article
```

**Review by helper-uttam — APPROVED** [review]
I've reviewed the article and made some changes. I loved the article it was great :)

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
please make the requested changed by @helper-uttam

**Comment by AmitKarnam** [comment]
I have made the all the changes suggested by @helper-uttam , Why does it say I dismissed  @helper-uttam 's stale review? When I made all the changes.

**Comment by helper-uttam** [comment]
> I have made the all the changes suggested by @helper-uttam , Why does it say I dismissed @helper-uttam 's stale review? When I made all the changes.

I think that's because I had approved the changes after reviewing and suggesting changes. I should I've requested changes but no worries :) I'll give it an approval once more xD

**Review by helper-uttam — CHANGES_REQUESTED** [review]
Need many changes eg:- line 23.
```diff
- clou- native
+ cloud-native 
```

**Comment by AmitKarnam** [comment]
I am closing this pull request as I messed up the changes part, I will be now opening a new PR shortly to handle this issue.

**Pull request closed without merging**